app/workers/evss/disability_compensation_form/job_status.rb

```diff
@@ -7,6 +7,7 @@ module JobStatus
       include SentryLogging
 
       def with_tracking(job_title, saved_claim_id, submission_id)
+        raise ArgumentError unless
```

Comment:
I'm not sure how this works (and tried googling to no avail). With no parameter for the `unless` statement, when will the `ArgumentError` be raised?

![unless](https://media.giphy.com/media/443fTQRDiAtAJHKGIm/giphy.gif)

Reply:
lol, yep my bad forgot the condition 😄